packages/@okta/vuepress-site/.vuepress/nav/guides.js

```diff
@@ -89,4 +89,10 @@ module.exports = [
       { title: 'Implement a Password Import Inline Hook', guideName: 'password-import-hook' },    
     ] 
   },
+  {
+    title: 'Migrate from Legacy Systems',
```

Comment:
We shouldn't have a category with only one member. @albertchen-okta suggested "User Management", since that's a category with a lot of potential for additional content.

Reply:
Are you saying to replace `Migrate from Legacy Systems` with `User Management`?

Reply:
@dogeared let's put this guide in the new "Deploy to Production" guide section that Susan just created